enry.py

```diff
@@ -0,0 +1,29 @@
+import ast
+import json
+import os
+import subprocess
+
+from tqdm import tqdm
+
+
+# 151.3629 if json.loads
+# 133.6954357624054 if ast.literal_eval
+def extract_languages(repo_path, json_path):
+    """
+    Extract programming language jsons into jsonl file
+    :param repo_path: path of extracted repository
+    :param json_path: path to jsonl file
+    """
+    path = os.walk(repo_path)
+    with open(json_path, "w") as json_file:
+        for root, _, files in tqdm(path):
+            for file in files:
+                enry_result = subprocess.check_output(
```

Comment:
Let's check OS to select `./enry` or `./enry.exe`
https://stackoverflow.com/questions/18472956/how-can-i-determine-if-the-operating-system-a-python-script-is-running-on-is-uni